tests/test_models.py

```diff
@@ -56,10 +56,10 @@ def test_create_transaction(session):
     session.refresh(transaction)
 
     assert transaction.id == 1
-    assert transaction.trans_date == datetime(2025, 6, 1, 15, 30, 0)
+    assert transaction.trans_date == date(2025, 6, 1)
```

Comment:
I don't think you have to mock dates here, but FYI:

[article] Freezegun - Real Joy for Fake Dates in Python
https://pybit.es/articles/guest-freezegun